src/components/common/VirtualizedSelectAutoSizer.ts

```diff
@@ -1,4 +1,4 @@
 // This module exists only to provide a convenient way the AutoSizer
 // used by VirtualizedSelect in unit tests. It is automatically mocked
-// in src/setupTests.js.
+// in src/setupTests.ts.
```

Comment:
Just cleanup because this file was converted to TypeScript in an earlier PR, but a couple of references were missed.